src/rust/ensogl/Cargo.toml

```diff
@@ -13,36 +13,36 @@ statistics = []
 no_unboxed_callbacks = []
 
 [dependencies]
-basegl-core-embedded-fonts = { version = "0.1.0"  , path = "embedded-fonts"   }
-basegl-core-msdf-sys       = { version = "0.1.0"  , path = "msdf-sys"         }
-enso-prelude               = { version = "0.1.0"  , path = "../prelude"       }
-basegl-system-web          = { version = "0.1.0"  , path = "../system/web"    }
-code-builder               = { version = "0.1.0"  , path = "../code-builder"  }
-data                       = { version = "0.1.0"  , path = "../data"          }
-enso-frp                   = { version = "0.1.0"  , path = "../frp"           }
-eval-tt                    = { version = "0.1.0"  , path = "../eval-tt"       }
-logger                     = { version = "0.1.0"  , path = "../logger"        }
-optics                     = { version = "0.1.0"  , path = "../optics"        }
-shapely                    = { version = "0.1.0"  , path = "../shapely/impl"  }
+basegl-core-embedded-fonts = { version = "0.1.0"  , path = "embedded-fonts"      }
```

Comment:
Shouldn't all occurrences of 'basegl' be changed to 'ensogl' ?

Reply:
good point. done